### ISSUE TITLE
cloudflare_access_service_token: Set duration when creating token

### ISSUE THREAD

**Issue by F21** [cloudflare/terraform-provider-cloudflare#2462]
### Current Terraform and Cloudflare provider version

Terraform v1.4.6
on linux_amd64
+ provider registry.terraform.io/cloudflare/cloudflare v4.6.0

### Description

It would be great if we can set the duration of the service token when creating it. Currently, it defaults to 1 year.

### Use cases

Depending on the security posture of the application, we would like to be able to set longer or shorter durations.

### Potential Terraform configuration

```hcl
resource "cloudflare_access_service_token" "my_app" {
  account_id = var.account_id
  name       = "CI/CD app"
  duration   = "10h"
}

```

**Comment by abatilo** [comment]
Has there been any progress on this?

**Comment by jacobbednarz** [comment]
closed by #2647
